api/logics.py

```diff
@@ -991,7 +991,7 @@ def is_penalized(user):
         return False, None
 
     @classmethod
-    def cancel_order(cls, order, user, state=None):
+    def cancel_order(cls, order, user, cancel_status=None):
```

Comment:
I used `cancel_status` because `current_status` was a bit confusing. Let me know if I should rename it.